pkg/metrics/otel.go

```diff
@@ -163,10 +163,15 @@ processors:
       # These labels are useful to users, but too noisy for global aggregation.
       - key: commit
         action: delete
-      - key: type
-        action: delete
   metricstransform/kubernetes:
     transforms:
+      - include: api_duration_seconds
+        action: update
+        operations:
+          # Eliminate 'type' tag from metric using aggregation
+          - action: aggregate_labels
+            label_set: [ status, operation ]
```

Comment:
remove leading and trailing space in the array

Reply:
Done!